apps/charterafrica/src/components/Mooc/Mooc.js

```diff
@@ -0,0 +1,100 @@
+import { Section } from "@commons-ui/core";
+import { Figure } from "@commons-ui/next";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import React from "react";
+
+import { neutral } from "@/charterafrica/colors";
+import LineClampedRichTypography from "@/charterafrica/components/LineClampedRichTypography";
+
+const Mooc = React.forwardRef(function Mooc(props, ref) {
+  const { title, link, image, sx } = props;
+
+  if (!title || !link || !image) {
+    return null;
+  }
+
```

Comment:
```suggestion
  if (!title) {
    return null;
  }
```

I think title is the most important one... the rest, you can just not render if not available but the message will still get through.